src/v2/Apps/__tests__/Fixtures/Order.ts

```diff
@@ -395,26 +395,14 @@ export const UntouchedBuyOrderWithShippingQuotes = {
             edges: [
```

Comment:
Below are changes the mocked data to reflect the new order we should receive quotes from Exchange API
The response should be ordered by `price_cents`:: https://github.com/artsy/exchange/pull/1156